runtime-light/stdlib/server/http-functions.cpp

```diff
@@ -181,3 +181,16 @@ void f$setrawcookie(const string &name, const string &value, int64_t expire_or_o
   }
   kphp::http::header({static_SB_spare.c_str(), static_SB_spare.size()}, false, kphp::http::status::NO_STATUS);
 }
+
+array<string> f$headers_list() noexcept {
```

Comment:
Maybe move it's implementation into the header?

Reply:
move